final_prep/annotations.rb

```diff
@@ -2,12 +2,15 @@
 # Try to focus on using correct technical vocabulary.
 # Use the # to create a new comment
 
-# Build a Bear
-
+# Define build_a_bear method with 5 arguments
 def build_a_bear(name, age, fur, clothes, special_power)
+  #define variable greeting to a string with interpolation of a method argument in it
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  #define variable with an array of two method arguments in it
   demographics = [name, age]
+  #define a variable with a string and interpolation of the method argument in it
   power_saying = "Did you know that I can #{special_power}?"
+  #Creat a built_bear hash that has basic info about the bear (object)
```

Comment:
You can go line by line here- it is worth noting where each key gets its value from.